src/framework/shortcuts/data/shortcuts.xml

```diff
@@ -677,6 +677,36 @@
     <seq>.</seq>
     <ctx>notation-staff-not-tab</ctx>
     </SC>
+  <SC>
+    <key>pad-rest</key>
+    <seq></seq>
+    <ctx>any</ctx>
```

Comment:
Please do let me know the context for these couple actions, I could add the ones nearby (like `notation-staff-not-tab` but I just wanted to be sure

Reply:
I believe this would get resolved by: https://github.com/musescore/MuseScore/pull/12414